bucket/ossutil.json

```diff
@@ -0,0 +1,48 @@
+{
+    "version": "1.7.14",
+    "description": "A user friendly command line tool to access AliCloud OSS.",
+    "homepage": "https://github.com/aliyun/ossutil",
+    "license": "MIT",
+    "notes": [
+        "Please follow the official documentation for ossutil at:",
+        "https://help.aliyun.com/document_detail/50451.html (for China users)",
+        "https://www.alibabacloud.com/help/en/object-storage-service/latest/developer-tools-ossutil (for international users)"
+    ],
+    "architecture": {
+        "64bit": {
+            "url": "https://gosspublic.alicdn.com/ossutil/1.7.14/ossutil64.zip",
+            "hash": "da2d170feaf30e70be93800e37a81a859279bfbbf75341fa7357ed549b679bb5",
+            "extract_dir": "ossutil64",
+            "bin": [
+                "ossutil64.exe",
```

Comment:
```suggestion
```

Reply:
I think there are people using platform-specific names, i.e. `ossutil64` and `ossutil32`, because you get these names if you just download and extract the package. I'm making these names available to users so as not to break their workflow if they're using platform-specific names (for example in their scripts).

Reply:
https://github.com/aliyun/ossutil#quick-use

The usage section uses name `ossutil`, and it's very simple to change the name from `ossutil64/32` to `ossutil`. I don't think there's need to keep these names, and since the 64bit and 32bit are not installed side by side, rename them should be okay.

Reply:
Done.